source/d1gfx.h

```diff
@@ -4,6 +4,8 @@
 #include <QMap>
 #include <QtEndian>
 
+#include <utility>
```

Comment:
I don't see any changes in this header that would make this necessary. Maybe it can be removed or moved to `d1gfx.cpp`?

Reply:
Done.